templates/about.html

```diff
@@ -1,66 +1,80 @@
+        Fortunately, some schools have a Syllabus repository like this already, but not nearly enough schools do it.
+        All of these are reasons why we made this tool and made it open-source for students of any university to adopt and improve.
+        If you can get your university to sponsor your fork of the app please do let us know and we'll redirect to yours.
+        Please feel free to add to the codebase or join the team if you feel inclined to help the effort.
+      </p>
+      <h2>The Team</h2>
+      <div class="row row-cols-md-2 row-cols-lg-3">
+        <div class="col my-2 text-center profile">
+          <img src="{% static 'vernon.jpg' %}" class="rounded-circle mb-2">
+          <h3><a class="text-body" href='https://www.linkedin.com/in/SyllabiShare'>Vernon Andrade</a></h3>
+          <h4>UVA CS '21</h4>
+          <h4>Lead Developer / Admin</h4>
+        </div>
+        <div class="col my-2 text-center profile">
+          <img src="{% static 'winston.jpg' %}" class="rounded-circle mb-2">
+          <h3><a class="text-body" href='https://www.linkedin.com/in/winston-liu'>Winston Liu</a></h3>
+          <h4>UVA CS '20</h4>
+          <h4>Developer</h4>
+        </div>
+        <div class="col my-2 text-center profile">
+          <img src="{% static 'unknown.png' %}" class="rounded-circle mb-2">
+          <h3><a class="text-body" href=''>Madison Flynn</a></h3>
+          <h4>UVA CS '21</h4>
+          <h4>Developer</h4>
+        </div>
+        <div class="col my-2 text-center profile">
+          <img src="{% static 'hamster.jpg' %}" class="rounded-circle mb-2">
+          <h3>Robbie B</h3>
+          <h4>UVA CS '26</h4>
+          <h4>Hamster Intern</h4>
+        </div>
+        <div class="col my-2 text-center profile">
+          <img src="{% static 'hamster.jpg' %}" class="rounded-circle mb-2" style="transform: scaleX(-1);">
+          <h3>Peter B</h3>
+          <h4>CNU '23</h4>
+          <h4>Hamster Documenter</h4>
+        </div>
+        <div class="col my-2 text-center profile">
+          <img src="{% static 'unknown.png' %}" class="rounded-circle mb-2">
+          <h3><a class="text-body" href=''>You!</a></h3>
+          <h4>UVA '2?</h4>
+          <h4>Bug Creator</h4>
+        </div>
+      </div>
+      <h2>Testimonials</h2>
+      <p>"I think it would benefit our students and potential students if more faculty members participated in this. I know that many student and friend groups keep copies of all syllabi and share them amongst themselves. I think posting them publicly on a site like this helps level the playing field for students who do not belong to those groups. It also seems like this would make my life easier, having students get my syllabi without dealing with all the personal requests through email. I'm glad you are working on this." ~ Prof. Raymond Pettit, UVA Computer Science</p>
```

Comment:
Just making sure that we do have consent to display this publicly?

Reply:
Yes I asked Pettit for permission!